packages/toolpad-core/src/themes/baseDarkTheme.ts

```diff
@@ -0,0 +1,129 @@
+import { createTheme } from '@mui/material/styles';
+
+const defaultDarkTheme = createTheme({
+  palette: {
+    mode: 'dark',
+  },
+});
+
+const baseDarkTheme = createTheme(defaultDarkTheme, {
+  palette: {
+    background: {
+      default: defaultDarkTheme.palette.grey['900'],
+    },
+    text: {
+      primary: defaultDarkTheme.palette.grey['100'],
+    },
+  },
+  typography: {
+    h6: {
+      fontWeight: '700',
+    },
+  },
+  components: {
```

Comment:
It feels wrong to me to add overrides in the theme. Some of these components are likely to appear in user code (Toolbar, List, IconButton). I don't think we should mandate any override to the user. Can't we style these components locally in the DashboardLayout?

Reply:
I guess the original idea was for these here to apply automatically also when the user uses these components... but let me see what I can do, maybe we can just scope these to the components used by the framework.